docs/impeller/Flutter-GPU.md

```diff
@@ -1,49 +1,36 @@
-**Flutter GPU** (previously referred to as "Dart GPU" or "Impeller Dart") is an effort to expose a low level graphics API in the Flutter Framework.
+**Flutter GPU** is a low level graphics API that ships as a package in the Flutter SDK. Flutter GPU enables you to build arbitrary renderers from stratch using just Dart and GLSL. No native platform code required.
 
-Design doc: https://flutter.dev/go/impeller-dart
+> [!Warning]
+> - Flutter GPU is in an early preview state and does not guarantee API stability.
+> - Flutter GPU currently requires [Impeller to be enabled](https://docs.flutter.dev/perf/impeller#availability).
+> - Automated shader building relies on the experimental Dart "Native Assets" feature.
```

Comment:
Consider linking "Native Assets" to https://github.com/dart-lang/sdk/issues/50565

Reply:
Done